src/mcts/node.h

```diff
@@ -286,6 +287,11 @@ class EdgeAndNode {
     return numerator * GetP() / (1 + GetNStarted());
   }
 
+  // Calculates score in centipawns
+  static float GetCpScore(const EdgeAndNode& edge) {
```

Comment:
That's fine. I use static methods when I don't modify the state of the object or need to override its behavior. Since this was a pure calculation I thought that was appropriate. I did make a mistake on my own terms here I would have just passed in the Q score as the parameter since there is no need for EdgeAndNode object at all.